command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
```

Comment:
In order to match our model, we need `update` and `show`. The former matches my previous comment, but the latter requires a tiny bit of additional thinking.

`list` is supposed to show you all possible entities and `show` all information about a particular entity. But the output of `list` will contain the section and option name (this pair forms the key) and the value. So `show` might appear useless. But we do have two timestamps in the database, so the client can return that. But I believe that consistency is good.

The additional commands don’t need to be implemented with the current PR; they can be put as feature requests.